.github/workflows/benchmark.yml

```diff
@@ -432,3 +432,22 @@ jobs:
           fi
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+  run_doc_benchmark:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout CredSweeper
+        if: ${{ 'pull_request' == github.event_name }}
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Send cURL request with the commit SHA
+        if: ${{ 'pull_request' == github.event_name }}
+        run: |
+          COMMIT_SHA=$(git rev-parse HEAD)
+          curl -X POST ${{ secrets.SLACK_URL}} \
+          --data-urlencode \
+          "payload={'text':'[BMT Request] ${{ github.event.repository.html_url }}/commit/${COMMIT_SHA}'}"
```

Comment:
perhaps, there is URL missing
```
curl: no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information
```

Reply:
The url is slack webhook url, so it should be encapsulated..
So I used repository -> secrets for hiding webhook url, and this can't be triggered from PR as I know.
Please check below.

https://stackoverflow.com/questions/73866908/github-actions-main-repository-secret-not-picked-up-from-pull-request-build